1001-2000/LIT1076GadlaTensaeWald.xml

```diff
@@ -5,9 +5,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1" xml:lang="gez">ገድለ፡ ተንስአ፡ ወልድ፡</title>
-            <title corresp="#t1" xml:lang="gez"  type="normalized">Gadla Tansǝʾa Wald</title>
-            <title corresp="#t1" xml:lang="en">Life of Tansǝʾa Wald</title>
+            <title xml:id="t1" xml:lang="gez">ገድለ፡ ተንስአ፡ ወልድ፡ ወመዓዛ፡ ድንግል </title>
+            <title corresp="#t1" xml:lang="gez"  type="normalized">Gadla Tansǝʾa Wald Wa-Maʿāzā Dǝngǝl</title>
+            <title corresp="#t1" xml:lang="en">Life of Tansǝʾa Wald and Maʿāzā Dǝngǝl</title>
+            <title xml:id="t2" xml:lang="gez">መጽሐፈ፡ አኅብሮ፡ ፡ድሙር፡ ገድሎሙ፡ ለአቡነ፡ ተንሥአ፡ ወልድ፡ ወመዓዛ ፡ድንግል </title>
+            <title corresp="#t2" xml:lang="gez"  type="normalized">Maṣḥafa ʾaḫbǝro gadlomu la-ʾAbuna Tansǝʾa Wald Wa-Maʿāzā Dǝngǝl</title>
```

Comment:
ድሙር is missing from the transcription, and wa- again would be better without capital